.travis.yml

```diff
@@ -1,16 +1,16 @@
 language: ruby
 cache: bundler
-sudo: false
```

Comment:
This has been deprecated for a while now.
https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration